engine/config.hpp

```diff
@@ -55,7 +55,9 @@ class user_node : public utils::config::typed_leaf_node< utils::passwd::user > {
 };
 
 
+/// \return A default test suite configuration.
```

Comment:
Why do these lines show up here and not next to the implementation, like in all other cases?

Reply:
Because I was being accidentally inconsistent. Fixed!